doc/code_style/mom.cpp

```diff
@@ -140,11 +136,7 @@ namespace parents {
 namespace mom {
 // <- A namespace does not increase the indentation level!
 
-YourMom::YourMom(int her_mass)
-	:
-	mass{her_mass},
-	number_of_moons{0},
-	stupidity{1} {
+YourMom::YourMom(int her_mass) : mass{her_mass}, number_of_moons{0}, stupidity{1} {
```

Comment:
the member initializers should remain on separate lines, complicated constructor calls and lambdas etc will be used for initialization, which becomes totally unreadable once that's only on one line.

Reply:
`ConstructorInitializerAllOnOneLineOrOnePerLine: 'false'`
`AllowAllConstructorInitializersOnNextLine: 'false'`

Reply:
that was due to me setting column limit to 100 changed it back to 80, didn't see that this was the standard before